release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -853,4 +860,30 @@ submodule openconfig-bgp-common {
   }
 
 
+  grouping auto-gen-link-bandwidth-config {
+    description
+      "Grouping of parameters related to automatic generation of
+      link-bandwidth extended community.";
+    leaf enabled {
+      description
+        "Generate link-bandwidth extnded community for BGP routes that do not
+        have link-bandwidth extnded community attached.";
+      type boolean;
+      default false;
+    }
+    leaf hold-down-time {
+      description
+        "Suppress frequent changes in the link-bandwidth value when bandwidth
+        increases.";
+      type uint16;
+    }
+    leaf transitive {
+      description
+        "Generated link-bandwidth attribute of transitive type. The non-transitive
+        is default and matches original draft.";
```

Comment:
I believe that [-11](https://www.ietf.org/archive/id/draft-ietf-idr-link-bandwidth-11.html) of the draft now supports creating transitive and non-transitive extcomms.